### PR TITLE
Major finals

### PR DESCRIPTION
**Pull request by kmlporto**

### CODE REVIEW

File: heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/IpsInterceptorService.java [hunk 1]
```diff
@@ -50,9 +51,9 @@
      * Checks if the request ip is in the whitelist
      *
      * @param whitelist Set of allowed ids
-     * @throws Throwable
+     * @throws IOException
      */
-    public void executeWhiteList(Set<String> whitelist) throws Throwable {
+    public void executeWhiteList(Set<String> whitelist) throws IOException {
```
Comment (smrafael) [inline heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/IpsInterceptorService.java:56]: Verify if this change is necessary

File: heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/util/ResponseHelper.java [hunk 2]
```diff
@@ -65,7 +66,7 @@ private ResponseHelper() {}
         return headers;
     }
 
-    public static String getResponseBody(RequestContext context, Map<String, String> headers) throws Throwable {
+    public static String getResponseBody(RequestContext context, Map<String, String> headers) throws IOException {
```
Comment (smrafael) [inline heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/util/ResponseHelper.java:69]: Verify if this change is necessary

File: heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/IpsInterceptorService.java (hunk 1 above)
Comment (kmlporto) [reply heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/service/IpsInterceptorService.java]: ok

File: heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/util/ResponseHelper.java (hunk 2 above)
Comment (kmlporto) [reply heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/util/ResponseHelper.java]: ok

**Pull request merged**